pkg/controller/user-cluster-controller-manager/resources/reconciler.go

```diff
@@ -857,6 +870,29 @@ func (r *reconciler) reconcileDeployments(ctx context.Context, data reconcileDat
 	return nil
 }
 
+func (r *reconciler) reconcileNetworkPolicies(ctx context.Context, data reconcileData) error {
+
+	namedNetworkPolicyCreatorGetters := []reconciling.NamedNetworkPolicyCreatorGetter{
+		kubesystem.DefaultNetworkPolicyCreator(),
+		coredns.KubeDNSNetworkPolicyCreator(data.clusterAddress.IP, int(data.clusterAddress.Port), data.k8sServiceApiIP.String()),
+	}
+
+	if r.userSSHKeyAgent {
+		namedNetworkPolicyCreatorGetters = append(namedNetworkPolicyCreatorGetters,
+			usersshkeys.NetworkPolicyCreator(data.clusterAddress.IP, int(data.clusterAddress.Port), data.k8sServiceApiIP.String()))
+	}
+
+	if r.isKonnectivityEnabled {
```

Comment:
haven't you forgotten about `konnectivity.NetworkPolicyCreator` here?

Reply:
@WeirdMachine ^